.github/workflows/beta.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
How does this work?

Reply:
it'll put a button "Run action" on the actions tab 😂

Reply:
![](https://i.imgur.com/rQvzdeg.gif)

Reply:
Nice ... how long has this been available? :D

Reply:
A couple of months...